src/world/map.c

```diff
@@ -3954,3 +3954,14 @@ bool map_surface_is_blocked(sint16 x, sint16 y){
 	return false;
 }
 
+/* Clears all map elements, to be used before generating a new map */
+void map_clear_all_elements()
+{
+	int mapMaxXY = RCT2_GLOBAL(RCT2_ADDRESS_MAP_MAX_XY, uint16);
```

Comment:
This variable is left unused, isn't it? Should it be used instead of the hardcoded `256`?